docs/docs/development/hardware-integration/new-shield.mdx

```diff
@@ -511,6 +511,28 @@ If all of your physical layouts use the same `kscan` node under the hood, you ca
 
 :::
 
+### Wired Split
+
+<SplitTabs>
+<TabItem value="unibody">This is only required for wired split keyboards.</TabItem>
+<TabItem value="split">
+
+If testing the experimental [wired split](../../features/split-keyboards.md) support, you should assign a [predefined](./pinctrl.mdx#predefined-nodes) or [pinctrl configured](./pinctrl.mdx) UART to the `device` property of a new node with `compatible` value of `"zmk,wired-split"`:
+
+```dts
+/ {
+    wired_split {
+        compatible = "zmk,wired-split";
+        device = <&pro_micro_serial>;
+    };
+};
+```
+
+See the [wired split](../../config/split.md#wired-split) configuration for more details.
+
```

Comment:
```suggestion

For wireless split keyboards, this step is **not** required and might want to be skipped, especially since the UART pins on your controller might already be in use for other functionality.
```

It doesn't hurt to be clearer here, since people blindly copy paste code blocks and Pro Micro UART pins are D1/D0, I believe.

Reply:
Open question whether we say "should be skipped" instead.

Reply:
> Open question whether we say "should be skipped" instead.

I think probably that wording is better, to be really clear.

Reply:
Added with the tweaked language.